internal/exec/exec_test.go

```diff
@@ -0,0 +1,64 @@
+package exec
+
+import (
+	"os"
+	"testing"
+	"unsafe"
+
+	"github.com/Microsoft/hcsshim/internal/winapi"
+	"golang.org/x/sys/windows"
+)
+
+func TestExecProcess(t *testing.T) {
```

Comment:
Would it be useful to have a test for when we're execing a process without additional proc attributes to verify that the changes you've made don't cause any problems there?

Reply:
Good call. I have this in my branch already, just need to push and figure out some other stuff

Reply:
added